examples/server/server.cpp

```diff
@@ -2591,11 +2592,22 @@ int main(int argc, char ** argv) {
         return false;
     };
 
-    auto middleware_server_state = [&res_error, &state](const httplib::Request &, httplib::Response & res) {
+    auto middleware_server_state = [&res_error, &state](const httplib::Request & req, httplib::Response & res) {
         server_state current_state = state.load();
         if (current_state == SERVER_STATE_LOADING_MODEL) {
-            res_error(res, format_error_response("Loading model", ERROR_TYPE_UNAVAILABLE));
-            return false;
+            httplib::Request & modified_req = (httplib::Request &) req;
```

Comment:
Dropping the `const` qualifier and mutating the request is not a valid solution.

Reply:
I simplified the diff and used res.set_content() to display a message with HTML.